text/survey_analysis_report2.md

```diff
@@ -0,0 +1,219 @@
+
+- Laboratory: Any organization that involves files from several projects. It can also be for a unique researchers, or for a consortium of laboratories.
+
+```
+
+## The experiment level
+
+
+We propose to keep data and code in different first level folders, 
+and to create several new folders (new data, analysis, and figure folders) for each new experiment.
+In addition, some of these new folders may also follow their own templates.
+For instance, some researchers could use a specific BIDS standard template for some imaging experiments.
+
+
+
+## The project and laboratory level
+
+We propose to share some sub-folders independently (for shared figures, report and conferences, and manuscripts) in order to be able to have them in the project folder or in a different folder structure merging information coming from different projects. One could for instance create a folder containing all manuscripts prepared in the lab (see Fig. 4).
+
+
+
+
+# Sharing and automation
+
+## Automation possibilities
+
+The creation of different sub-folders for one experiment could easily be automated in your computer language of choice. An automation would make sure that the folder names are kept consistent for each experiment.
+
+Working on the laboratory level is more complex. If one wants to have cross-folder organization locally, one can work with "alias folders", where the user can create shortcuts to specific folders using a different organization. The data exists only once and there is no issues of synchronization. 
+
+The expected use case is different though, and we expect some users to have certain files organized by projects and other users having the same files organised by file type. This requires to share sub-folders independently and set the different instance of these sub-folders to be linked together. This is pretty complex to set up using common cloud technology (dropbox-like) that cannot be easily automated. As explained below, the open source GIN-tonic tool allows to set it up automatically, using the git submodule technology.
+
+## GIN-tonic implementation
+
+GIN is the G-Node infrastructure.
+It is based on gogs, git and git annex technologies and
+ brings non only most of the project management and coordination tools that made the success of open source software development,
+but also large file support and data publication.
+It is compatible with the git submodule technology, where sub-folders can be synchronized, shared and published independently of the other sub-folders,
+while looking completely normal on one's computer.
+GIN brings therefore the possibility to publish sub-modules independently of each others, 
+which will ease the opening of research data.
+It might also make [the use of markdown and latex to write manuscripts](https://oa-pub.hos.tuhh.de/en/project/) a straightforward choice, as these technologies can use git as a version control system.
```

Comment:
```suggestion
It might also make [the use of markdown and LaTex to write manuscripts](https://oa-pub.hos.tuhh.de/en/project/) a straightforward choice, as these technologies can use git as a version control system.
```